### PR TITLE
RHOAIENG-11046 - Add support for AMD GPU image

### PR DESCRIPTION
**Pull request by jiripetrlik**
# Issue link

# What changes have been made

# Verification steps

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: support/defaults.go [hunk 1]
```diff
@@ -8,4 +8,5 @@ const (
 	CodeFlareSDKVersion = "v0.20.2"
 	RayVersion          = "2.35.0"
 	RayImage            = "quay.io/modh/ray:2.35.0-py39-cu121"
+	RayAMDGpuImage      = "quay.io/rhoai/ray:2.35.0-py39-rocm61-torch24-fa26"
```
Comment (ChughShilpa) [inline support/defaults.go:11]: This rocm-ray image `quay.io/rhoai/ray:2.35.0-py39-rocm61-torch24-fa26` is torch compatible.
IMO, we should use base Ray AMD/ROCm image"2.35.0-py39-rocm61"  built with konflux
`quay.io/modh/ray:2.35.0-py39-rocm61`

**Comment by jiripetrlik** [comment]
Hello @anishasthana can you please help us to find out what is the right image?

**Comment by astefanutti** [comment]
@jiripetrlik Yes that's preferable to default to `quay.io/modh/ray:2.35.0-py39-rocm61` as @ChughShilpa suggested.

**Review by sutaakar — COMMENTED** [review]
/lgtm

**Review by ChughShilpa — APPROVED** [review]

File: support/environment.go [hunk 2]
```diff
@@ -78,6 +79,10 @@ func GetRayImage() string {
 	return lookupEnvOrDefault(CodeFlareTestRayImage, RayImage)
 }
 
+func GetRayAMDGpuImage() string {
```
Comment (astefanutti) [inline support/environment.go:82]: I'd suggest to call it `GetRayROCmImage`, and maybe rename `CodeFlareTestRayImage` to `CodeFlareTestRayCUDAImage`.

**Review by astefanutti — APPROVED** [review]

**Pull request merged**